Cargo.toml

```diff
@@ -73,6 +73,7 @@ tracing-subscriber = "0.3.7"
 yoke = { version = "0.7.2", features = ["derive"] }
 zip = { version = "1.1.1", default-features = false }
 metal = { version = "0.27.0", features = ["mps"]}
+either = { version = "1.13.0", features = ["serde"] }
```

Comment:
Let's not introduce new dependencies please. Especially here it's pretty straightforward to do without it.

Reply:
I removed the either dep throughout.